Index: sessions.c
===================================================================
RCS file: /tcpdump/master/tcpslice/sessions.c,v
retrieving revision 1.1
diff -u -r1.1 sessions.c
--- sessions.c	30 Nov 2006 17:26:47 -0000	1.1
+++ sessions.c	23 Jan 2008 17:48:15 -0000
@@ -775,10 +775,10 @@
     return 0;
   if (osip_strcasecmp(ctt->subtype, "sdp"))
     return 0;
-  for (i = 0; !osip_list_eol(msg->bodies, i); ++i) {
+  for (i = 0; !osip_list_eol(&(msg->bodies), i); ++i) {
     sdp = NULL;
     sdp_message_init(&sdp);
-    tmp = ((osip_body_t *)osip_list_get(msg->bodies, i))->body;
+    tmp = ((osip_body_t *)osip_list_get(&(msg->bodies), i))->body;
     if (sdp_message_parse(sdp, tmp)) {
       sdp_message_free(sdp);
       continue;
